WooCommerce/Classes/ViewRelated/Feature Highlight/Tooltip.swift

```diff
@@ -0,0 +1,420 @@
+import UIKit
+
+final class Tooltip: UIView {
+    static let arrowWidth: CGFloat = 17
```

Comment:
nit: does it need to be public?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/355adfad453717bd75f09d5f6008dad331b5e10f